Makefile

```
@@ -272,6 +273,8 @@ run-re-frisk: ##@run Start re-frisk server
 
 # TODO: Migrate this to a Nix recipe, much the same way as nix/mobile/android/targets/release-android.nix
 run-android: export TARGET := android
+# INFO: If it's empty (no devices attached, parsing issues, script error) - for Nix it's the same as not set.
+run-android: export ANDROID_ABI_INCLUDE := $(CONNECTED_ANDROID_DEVICES_ABIS)
```

I don't think we need a separate variable for this if we just use it in one place.
```suggestion
run-android: export ANDROID_ABI_INCLUDE ?= $(shell ./scripts/adb_devices_arch.sh)
```
Also I'd use `?=` lazy assignment instead of `:=` so people can override this with env vars.